src/generator.ts

```diff
@@ -258,12 +258,18 @@ export class Generator {
     let gitIgnore = '';
     if (fs.existsSync(gitIgnorePath))
       gitIgnore = fs.readFileSync(gitIgnorePath, 'utf-8').trimEnd() + '\n';
-    if (!gitIgnore.includes('node_modules'))
-      gitIgnore += 'node_modules/\n';
-    gitIgnore += '/test-results/\n';
-    gitIgnore += '/playwright-report/\n';
-    gitIgnore += '/blob-report/\n';
-    gitIgnore += '/playwright/.cache/\n';
+    const valuesToAdd = { 
+      'node_modules/': /^node_modules\/?/m,
+      '/test-results/': /^test-results\/?$/m,
```

Comment:
```suggestion
      '/test-results/': /^\/test-results\/?$/m,
```

Reply:
I think the reason I didn't suggest this change was that the following duplicate line in `.gitignore` would be generated (2nd line not needed):

```
test-results/
/test-results/
```

Reply:
But it's also not such a big deal to have this duplicate line, so Max's suggestion above could also just be applied.

Reply:
`'/test-results/': /^\/?test-results\/?$/m,` sounds good as well.